src/Resources/Transactions/Operations/ListTransactions.php

```diff
@@ -34,6 +36,7 @@ public function __construct(
         private readonly array $statuses = [],
         private readonly array $subscriptionIds = [],
         private readonly ?DateComparison $updatedAt = null,
+        private readonly array $origin = [],
```

Comment:
following on from the other variables here should we make this plural ?

Reply:
Good catch, updated ✅